client/windows/settings/SettingsMainWindow.cpp

```diff
@@ -57,7 +56,7 @@ SettingsMainWindow::SettingsMainWindow(BattleInterface * parentBattleUi) : Inter
 	std::shared_ptr<CButton> restartButton = widget<CButton>("restartButton");
 	assert(restartButton);
 
-	if(CSH->isGuest())
+	if(CSH->isGuest() || parentBattleUi)
 	{
 		loadButton->block(true);
```

Comment:
loading game during battle works, no reason to block it - it is wanted feature

Reply:
Enabled loading, kept restart & save locked